apps/backend/src/services/lib/item.service.js

```diff
@@ -265,6 +265,19 @@ const updateItem = async (id, updateData, space, block) => {
     return updatedItem;
 };
 
+const deleteItem = async (id, space, block, user) => {
+    const deletedItem = await Item.findOneAndDelete({
```

Comment:
To remove an item from the database, set isDeleted to true, so user can undo the deletion.